src/compas/scene/sceneobject.py

```diff
@@ -123,29 +165,69 @@ def __init__(
     def __data__(self):
         # type: () -> dict
         return {
-            "item": str(self.item.guid),
-            "settings": self.settings,
-            "children": [child.__data__ for child in self.children],
+            "item": self._item,
+            "name": self.name,
+            "color": self.color,
+            "opacity": self.opacity,
+            "show": self.show,
+            "transformation": self.transformation,
         }
 
-    @classmethod
-    def __from_data__(cls, data):
-        # type: (dict) -> None
-        raise TypeError("Serialisation outside Scene not allowed.")
-
     def __repr__(self):
         # type: () -> str
         return "<{}: {}>".format(self.__class__.__name__, self.name)
 
     @property
     def scene(self):
         # type: () -> compas.scene.Scene | None
-        return self.tree
+        return self._scene
 
     @property
     def item(self):
         # type: () -> compas.data.Data
-        return self._item
+        return self.scene.datastore[self._item]
```

Comment:
`item` now becomes a getter that retrieve the live object from datastore, what we actually store as `_item` is the `guid`